docs/docusaurus/docs/components/LinkCard/index.js

```diff
@@ -1,28 +0,0 @@
-import React from 'react';
```

Comment:
Are these deleted files all identical between versions. Is there a shared copy they are now using?
My only concern would be introducing a dependency on the versions with each other and I'm curious if this would introduce any problems.

Reply:
yup! so within `docusaurus/docs/` we have `src/components/LinkCard` (and the grid). `diff` shows no difference between either the versioned directories or the src one.

Since this is more global styling than anything version, I think it makes sense to live external to any versions (src). (https://docusaurus.io/docs/2.x/versioning#use-absolute-import-within-the-docs provides an example of this).

Reply:
Sounds good to me.